### ISSUE TITLE
Pretrained model of AnyNet

### ISSUE THREAD

**Issue by chenyilun95** [mileyan/AnyNet#8]
Thanks for releasing the code!

Could you please provide the pre-trained model of AnyNet in SceneFlow dataset ? Thank you a lot !

**Comment by yxchng** [comment]
@mileyan Hi, will it be released?

**Comment by abhishekmaha23** [comment]
Watching!

**Comment by mileyan** [comment]
Has released the pretrained model. Please check it. Thanks.

**Comment by ghost** [comment]
@mileyan   It looks like the pre-trained model is trained with spn, so one would have to compile the spn module which with pytorch 1.0+ is not possible (without some re-write to C++ extensions).

Any plans to release a pre-trained model without spn? Or do you know anyone who has gotten spn to work with pytorch 1.0+?

**Comment by mileyan** [comment]
Hi,

> @mileyan It looks like the pre-trained model is trained with spn, so one would have to compile the spn module which with pytorch 1.0+ is not possible (without some re-write to C++ extensions).
> 
> Any plans to release a pre-trained model without spn? Or do you know anyone who has gotten spn to work with pytorch 1.0+?

Now our code supports pytorch 1.0. Please check it.

**Comment by youmi-zym** [comment]
Hi,
Thanks for sharing the pre-trained checkpoint on SceneFlow, can you specify the metric you got? Such as EPE, 3PX and so on. I want to make sure my reimplementation approaching your result. Thanks very much.

**Comment by mileyan** [comment]
Hi, we evaluate the 3-pixel error on KITTI dataset.

**Comment by youmi-zym** [comment]
Hi @mileyan, How about metric on SceneFlow?

**Comment by mileyan** [comment]
Hi, we evaluate 3-pixel error on KITTI set. We randomly split the kitti training set 4 times and the ratio of training vs validation is 4:1. The result in the paper is the mean of results. 
In SceneFlow, yes, we do end point error.

**Comment by youmi-zym** [comment]
Hi @mileyan, thanks for detailing. As for SceneFlow, I have trained for 10 epoch and got EPE=3.377. Is that approaching your result? Btw, I have also extended the training schedule for 20 epoch with a constant learning rate and got EPE=3.214.  Can you give the specific EPE you got, thanks a lot.

**Comment by mileyan** [comment]
Yes, the result is close to 3.377. It is very interesting to see that training more epochs can get better performance.

**Comment by youmi-zym** [comment]
Well, thanks a lot.
